docs/wiki/deployment/remote.md

```diff
@@ -239,7 +239,7 @@ We will use an **osqueryd** client and set the required TLS settings. When enfor
 ```
 $ osqueryd --verbose --ephemeral --disable_database \
     --tls_hostname localhost:8080 \
-    --tls_server_certs ./tools/tests/test_server_ca.pem \
+    --tls_server_certs ./tools/tests/test_server.pem \
```

Comment:
I think it's correct the way it is. I just ran the tests locally, and with your changes it fails:
```
osquery git:master ❯ osqueryd --verbose --ephemeral --disable_database \                                                                                                                                                                    ⏎
--tls_hostname localhost:8080 \
--tls_server_certs ./tools/tests/test_server.pem \
--config_plugin tls \
--config_tls_endpoint /config \
--logger_tls_endpoint /logger \
--logger_plugin tls  \
--enroll_tls_endpoint /enroll \
--enroll_secret_path ./tools/tests/test_enroll_secret.txt
I0205 22:37:18.918870 2398614336 init.cpp:380] osquery initialized [version=3.0.0-11-g3decac94]
I0205 22:37:18.920620 2398614336 extensions.cpp:346] Found autoloadable extension: /usr/local/share/osquery/tables/corp_tables.ext
I0205 22:37:18.920711 2398614336 extensions.cpp:346] Found autoloadable extension: /usr/local/share/osquery/tables/corp_python_tables.ext
I0205 22:37:18.922350 31121408 watcher.cpp:549] osqueryd watcher (35410) executing worker (35411)
I0205 22:37:18.922951 31121408 watcher.cpp:591] Created and monitoring extension child (35412): /usr/local/share/osquery/tables/corp_python_tables.ext
I0205 22:37:18.923622 31121408 watcher.cpp:591] Created and monitoring extension child (35413): /usr/local/share/osquery/tables/corp_tables.ext
I0205 22:37:18.937892 2398614336 init.cpp:383] osquery extension initialized [sdk=3.0.0]
I0205 22:37:18.944803 2398614336 init.cpp:377] osquery worker initialized [watcher=35410]
E0205 22:37:18.946105 2398614336 init.cpp:611] Cannot start extension manager: Cannot write extension socket: /var/osquery/osquery.em
I0205 22:37:18.946751 2398614336 tls_enroll.cpp:63] TLSEnrollPlugin requesting a node enroll key from: https://localhost:8080/enroll
I0205 22:37:18.965915 2398614336 tls.cpp:198] TLS/HTTPS POST request to URI: https://localhost:8080/enroll
W0205 22:37:18.993242 2398614336 tls_enroll.cpp:70] Failed enrollment request to https://localhost:8080/enroll (Request error: certificate verify failed) retrying...
I0205 22:37:20.003669 2398614336 tls.cpp:198] TLS/HTTPS POST request to URI: https://localhost:8080/enroll
W0205 22:37:20.006961 2398614336 tls_enroll.cpp:70] Failed enrollment request to https://localhost:8080/enroll (Request error: certificate verify failed) retrying...
```
However with the way it currently is written it enrolls correctly and we see results coming back:
```
osquery git:master ❯ osqueryd --verbose --ephemeral --disable_database \
> --tls_hostname localhost:8080 \
> --tls_server_certs ./tools/tests/test_server_ca.pem \
> --config_plugin tls \
> --config_tls_endpoint /config \
> --logger_tls_endpoint /logger \
> --logger_plugin tls  \
> --enroll_tls_endpoint /enroll \
> --enroll_secret_path ./tools/tests/test_enroll_secret.txt
I0205 22:36:41.528208 2398614336 init.cpp:380] osquery initialized [version=3.0.0-11-g3decac94]
I0205 22:36:41.529656 2398614336 extensions.cpp:346] Found autoloadable extension: /usr/local/share/osquery/tables/corp_tables.ext
I0205 22:36:41.529784 2398614336 extensions.cpp:346] Found autoloadable extension: /usr/local/share/osquery/tables/corp_python_tables.ext
I0205 22:36:41.531455 212459520 watcher.cpp:549] osqueryd watcher (35307) executing worker (35308)
I0205 22:36:41.531915 212459520 watcher.cpp:591] Created and monitoring extension child (35309): /usr/local/share/osquery/tables/corp_python_tables.ext
I0205 22:36:41.532393 212459520 watcher.cpp:591] Created and monitoring extension child (35310): /usr/local/share/osquery/tables/corp_tables.ext
I0205 22:36:41.550861 2398614336 init.cpp:383] osquery extension initialized [sdk=3.0.0]
I0205 22:36:41.560925 2398614336 init.cpp:377] osquery worker initialized [watcher=35307]
E0205 22:36:41.562572 2398614336 init.cpp:611] Cannot start extension manager: Cannot write extension socket: /var/osquery/osquery.em
I0205 22:36:41.563392 2398614336 tls_enroll.cpp:63] TLSEnrollPlugin requesting a node enroll key from: https://localhost:8080/enroll
I0205 22:36:41.581010 2398614336 tls.cpp:198] TLS/HTTPS POST request to URI: https://localhost:8080/enroll
I0205 22:36:41.594084 2398614336 tls.cpp:198] TLS/HTTPS POST request to URI: https://localhost:8080/config
I0205 22:36:41.598665 2398614336 events.cpp:825] Event publisher not enabled: openbsm: Publisher disabled via configuration
I0205 22:36:41.598742 2398614336 events.cpp:825] Event publisher not enabled: scnetwork: Publisher not used
I0205 22:36:41.598768 2398614336 events.cpp:825] Event publisher not enabled: event_tapping: Publisher disabled via configuration
I0205 22:36:41.621680 52363264 events.cpp:746] Starting event publisher run loop: diskarbitration
I0205 22:36:41.621696 52899840 events.cpp:746] Starting event publisher run loop: fsevents
I0205 22:36:41.621704 2398614336 main.cpp:106] Not starting the distributed query service: Distributed query service not enabled.
I0205 22:36:41.621721 53436416 events.cpp:746] Starting event publisher run loop: iokit
I0205 22:36:41.622113 55046144 scheduler.cpp:75] Executing scheduled query tls_proc: select * from processes
I0205 22:36:41.914880 55046144 query.cpp:106] Storing initial results for new scheduled query: tls_proc
I0205 22:36:41.918658 55046144 scheduler.cpp:140] Found results for query: tls_proc
I0205 22:36:42.933285 55046144 scheduler.cpp:75] Executing scheduled query tls_proc: select * from processes
I0205 22:36:43.246049 55046144 scheduler.cpp:140] Found results for query: tls_proc
I0205 22:36:44.250792 55046144 scheduler.cpp:75] Executing scheduled query tls_proc: select * from processes
I0205 22:36:44.567343 55046144 scheduler.cpp:140] Found results for query: tls_proc
E0205 22:36:44.940454 2398614336 example_extension.cpp:149] Extension socket not available: /var/osquery/osquery.em
I0205 22:36:45.574481 55046144 scheduler.cpp:75] Executing scheduled query tls_proc: select * from processes
I0205 22:36:45.887439 55046144 scheduler.cpp:140] Found results for query: tls_proc
I0205 22:36:45.933631 51826688 tls.cpp:198] TLS/HTTPS POST request to URI: https://localhost:8080/logger
I0205 22:36:46.100191 51826688 tls.cpp:198] TLS/HTTPS POST request to URI: https://localhost:8080/logger
```